--- install.orig.sh     2026-05-24 18:55:17.595056901 -0400
+++ install.sh  2026-05-24 19:02:42.746128439 -0400
@@ -158,6 +158,7 @@
 # Test whether dependencies are present.  Returns nonzero if any dependency is
 # missing.
 function hasDependencies() {
+    return 0 # Bazzite have everything except xterm, needs patcher script to update anyway so who cares
     # Check for libxkbcommon-x11, libxkbcommon, and libxcb-xkb
 
     # Wrap each test in `if ...; then return 1; fi` to play nicely with set -e
@@ -298,21 +299,22 @@
     # Ideally we don't want to put them in pixmaps but this seems to be the
     # best place for cross platform usage, and is part of the XDG spec
     # Some platforms like Arch do't have this path
-    if [ ! -d "/usr/share/pixmaps" ]; then
-        sudo mkdir -p /usr/share/pixmaps/
+    if [ ! -d "$HOME/.local/share/pixmaps" ]; then
+        mkdir -p $HOME/.local/share/pixmaps/
     fi
-    sudo cp "$root/installfiles/app-icon.png" "/usr/share/pixmaps/${brandCode}vpn.png"
-    [ -e "/usr/share/pixmaps/${brandCode}.png" ] && sudo rm -rf "/usr/share/pixmaps/${brandCode}.png"
+    cp "$root/installfiles/app-icon.png" "$HOME/.local/share/pixmaps/${brandCode}vpn.png"
+    [ -e "$HOME/.local/share/pixmaps/${brandCode}.png" ] && rm -rf "$HOME/.local/share/pixmaps/${brandCode}.png"
     echoPass "Installed icon"
 
-    if [ ! -d "/usr/share/applications" ]; then
-        sudo mkdir -p /usr/share/applications/
+    if [ ! -d "$HOME/.local/share/applications" ]; then
+        mkdir -p $HOME/.local/share/applications/
     fi
-    sudo cp "$root/installfiles/piavpn.desktop" "/usr/share/applications/${brandCode}vpn.desktop"
+    cp "$root/installfiles/piavpn.desktop" "$HOME/.local/share/applications/${brandCode}vpn.desktop"
     if hash update-desktop-database 2>/dev/null; then
         # Silence output from update-desktop-database; it dumps errors for _any_
         # quirks in installed .desktop files, not just ours.
-        sudo update-desktop-database 2>/dev/null
+        update-desktop-database "$HOME/.local/share/applications/" 2>/dev/null \
+            || echoFail "Failed to update desktop database, application shortcut may not work"
     fi
     echoPass "Created desktop entry"
 
